src/libslic3r/AppConfig.hpp

```diff
@@ -22,6 +22,7 @@ using namespace nlohmann;
 #define ENV_PRE_HOST		"2"
 #define ENV_PRODUCT_HOST	"3"
 
+#define SUPPORT_REVERSE_MOUSE_ZOOM
```

Comment:
Since we have reverse_mouse_wheel_zoom option, the macro seems unnecessary, can we remove it?

Reply:
That was my initial thought: just delete the preprocessor guard around that feature. 
My second thought was that adding single line of a defined macro may reduce complexity during possible upmerge in the future (if there will be any).

I will delete that macro altogether as I prefer this way too.

Reply:
Done